src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -330,7 +339,121 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+                    },
+                    preloadResources: "prefsEditor",
+                    listeners: {
+                        "onCreate.loadResources": {
+                            listener: "fluid.prefs.separatedPanel.lazyLoad.preloadResources",
+                            args: ["{that}", {expander: {func: "{that}.resolveResources"}}, "{that}.options.preloadResources"]
+                        },
+                        "{separatedPanel}.events.onLazyLoad": {
+                            listener: "fluid.resourceLoader.loadResources",
+                            args: ["{messageLoader}", {expander: {func: "{messageLoader}.resolveResources"}}]
+                        }
+                    }
+                }
+            },
+            slidingPanel: {
+                options: {
+                    invokers: {
+                        operateShow: {
+                            funcName: "fluid.prefs.separatedPanel.lazyLoad.showPanel",
+                            args: ["{separatedPanel}", "{that}.events.afterPanelShow.fire"]
+                        }
+                    }
+                }
+            }
+        }
+    });
+
+    fluid.prefs.separatedPanel.lazyLoad.showPanel = function (separatedPanel, callback) {
+        if (separatedPanel.prefsEditor) {
+            fluid.invokeLater(callback);
+        } else {
+            separatedPanel.events.onReady.addListener(function (that) {
+                that.events.onReady.removeListener("showPanelCallBack");
+                fluid.invokeLater(callback);
+            }, "showPanelCallBack");
+            separatedPanel.events.onLazyLoad.fire();
+        }
+
+    };
+
+    /**
+     * Used to override the standard "onCreate.loadResources" listener for fluid.resourceLoader component,
+     * allowing for pre-loading of a subset of resources. This is requried for the lazyLoading workflow
+     * for the "fluid.prefs.separatedPanel.lazyLoad".
+     *
+     * @param {Object} that - the component
+     * @param {Object} resource - all of the resourceSpecs to load, including preload and others.
+     *                            see: fluid.fetchResources
+     * @param {Array/String} toPreload - a String or an Array of Strings corresponding to the names
+     *                                   of the resources, supplied in the resource argument, that
+     *                                   should be loaded. Only these resources will be loaded.
+     */
+    fluid.prefs.separatedPanel.lazyLoad.preloadResources = function (that, resources, toPreload) {
+        toPreload = fluid.makeArray(toPreload);
+        var preloadResources = {};
+
+        fluid.each(toPreload, function (resourceName) {
+            preloadResources[resourceName] = resources[resourceName];
+        });
+
+        fluid.fetchResources(preloadResources, function () {
+            that.resources = preloadResources;
```

Comment:
Ok. Could you add a comment indicating that this logic is copied from the implementation of fluid.resourceLoader.loadResources